tools/cmake_call.sh

```diff
@@ -13,12 +13,19 @@ cd src
 sh ./scripts/nlopt_download.sh ${RSCRIPT_BIN}
 dot() { file=$1; shift; . "$file"; }
 dot ./scripts/r_config.sh ""
+
+# Only R > 4 will have the variables
+$RSCRIPT_BIN --vanilla -e "getRversion() > 4" | grep TRUE > /dev/null
+if [ $? -eq 0 ]; then
+	NEW_R="-D CMAKE_AR=${AR} -D CMAKE_RANLIB=${RANLIB} "
+else
+	NEW_R=""
+fi
```

Comment:
A form I have seen more often is to set two vars directly such as, say, `CMAKE_ADD_AR` and have that be `CMAKE_ADD_AR="-D CMAKE_AR=$(AR)"` or else `CMAKE_ADD_AR=""` and include those below.  A little easier to read six months from now when we will have forgotten the issue.